app/validators/sections/section_validator.py

```diff
@@ -292,3 +295,121 @@ def _get_question_variant_fields_sets(variants):
                 results["number_of_answers"].add(len(results["answer_ids"]))
 
         return results
+
+    def validate_number_of_lists(self):
+        if self.has_list_summary_with_non_item_answers() and self.has_multiple_lists():
+            self.add_error(error_messages.MULTIPLE_LISTS)
+
+    def validate_number_of_list_collectors(self):
+        if (
+            self.has_list_summary_with_non_item_answers()
+            and self.has_multiple_list_collectors()
+        ):
+            self.add_error(error_messages.MULTIPLE_LIST_COLLECTORS)
+
+    def has_list_summary_with_non_item_answers(self):
+        if summary := self.schema_element.get("summary"):
+            show_non_item_answers = summary.get("show_non_item_answers")
+            if summary.get("items") and show_non_item_answers is True:
+                return True
+
+    def has_multiple_list_collectors(self):
+        list_collectors = []
+        for group in self.schema_element.get("groups"):
+            list_collectors.extend(
+                block
+                for block in group.get("blocks")
+                if block["type"] in ["ListCollector"]
+            )
+
+        return len(list_collectors) > 1
+
+    def has_multiple_lists(self):
+        lists = []
+        for group in self.schema_element.get("groups"):
+            lists.extend(
+                block.get("for_list")
+                for block in group.get("blocks")
+                if block["type"] in ["ListCollector"]
+            )
+
+        return len(lists) > 1
```

Comment:
Both of these tell you the same thing, you only need 1 of them?. The `has_multiple_lists` check can be dropped as if we verify you only have 1 list collector, then that implies you also only have 1 list.